SpatialGDK/Source/SpatialGDK/Public/Utils/SpatialActorUtils.h

```diff
@@ -136,4 +138,58 @@ inline bool IsDynamicSubObject(const USpatialNetDriver& NetDriver, const AActor&
 	return !ActorClassInfo.SubobjectInfo.Contains(SubObjectOffset);
 }
 
+using FSubobjectToOffsetMap = TMap<UObject*, ObjectOffset>;
+static FSubobjectToOffsetMap CreateOffsetMapFromActor(const USpatialNetDriver& NetDriver, AActor& Actor)
+{
+	USpatialPackageMapClient& PackageMap = *NetDriver.PackageMap;
+
+	const FClassInfo& ActorInfo = NetDriver.ClassInfoManager->GetOrCreateClassInfoByClass(Actor.GetClass());
```

Comment:
@improbable-dmitrii Do you think it's better to pass the NetDriver here and extract information from it, or pass in a packageMap reference and ActorInfo reference directly?

Reply:
I'd advocate for the second approach, no need to couple the NetDriver any tighter if we can help it.

Reply:
changed to second approach!